tests/testthat/test-draw_heatmap.R

```diff
@@ -0,0 +1,10 @@
+test_that("draw_heatmap works as expected", {
+  result <- draw_heatmap(
+    hermes_data,
```

Comment:
the heatmap is too large to see anything, please use a subset of genes/samples (similar as you did in the design doc)